plugin/storage/kafka/options.go

```diff
@@ -74,15 +83,37 @@ func (opt *Options) AddFlags(flagSet *flag.FlagSet) {
 		defaultEncoding,
 		fmt.Sprintf(`Encoding of spans ("%s" or "%s") sent to kafka.`, EncodingJSON, EncodingProto),
 	)
+	flagSet.Int(
+		configPrefix+suffixRequiredAcks,
+		defaultRequiredAcks,
+		"(experimental) Required kafka broker acknowledgement. default = 1, no response = 0, wait for local = 1, wait for all = -1",
+	)
+	flagSet.Int(
+		configPrefix+suffixCompression,
+		defaultCompression,
+		"(experimental) Type of compression to use on messages.  none = 0, gzip = 1, snappy = 2, lz4 = 3, zstd = 4",
+	)
+	flagSet.Int(
+		configPrefix+suffixCompressionLevel,
+		defaultCompressionLevel,
+		"(experimental) Level of compression to use on messages. default= -1000, gzip = 1-9 (default = 6), snappy = none, lz4 = 1-17 (default = 9), zstd = -131072 - 22 (default = 3)",
```

Comment:
the numbers make almost no sense. If it's a "level of compression", can we normalize it to a single scale, e.g. 1 to 9 (lowest - highest)?

Reply:
why is default -1000?  

also, Viper already  prints the default value in parantheses

Reply:
done